src/deepness/common/processing_parameters/standardization_parameters.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+
+
+class StandardizationParameters:
+    mean: float
```

Comment:
why mean/std fields here, if they are assigned to self? Should be removed probably